packages/web/src/lib/wallectconnect-mode.ts

```diff
@@ -196,20 +196,10 @@ export class WallectConnectSession {
 	private async waitForConfirmation(
 		txId: string
 	): Promise<algosdk.modelsv2.PendingTransactionResponse> {
```

Comment:
yeah, I told that in comment, that's `AlgoSigner` wallet and it has its own `algod` function which provides transaction status , so I don't think we can use `algosdk.waitForConfirmation` in that. If it can be done, please let me know.

Reply:
> yeah, I told that in comment, that's `AlgoSigner` wallet and it has its own `algod` function which provides transaction status , so I don't think we can use `algosdk.waitForConfirmation` in that. If it can be done, please let me know.

Sorry, I didn't check the web-mode.ts using `AlgoSigner`. Thanks for your inform!